packages/react/src/components/Dropdown/DropdownItem.tsx

```diff
@@ -19,10 +19,10 @@ export interface DropdownItemProps extends UIComponentProps<DropdownItemProps> {
   active?: boolean
 
   /** Item's content. */
-  content?: string
+  content?: ShorthandValue
```

Comment:
pls also export:
```typescript
export DropdownItem.slotClassNames = {
  content: `${DropdownItem.className}__content`,
  header: `${DropdownItem.className}__header`,
}
```
same as it's done here: https://github.com/stardust-ui/react/blob/7bd74262315cdc03679662be8551b9fa5e3b9eb4/packages/react/src/components/Dropdown/Dropdown.tsx#L1104

There are users that reported that they cannot target the header/content in order to style them so this PR is the best opportunity to add this. Otherwise, pls. let me know once it's fixed and I'll create issue + PR

fyi @mnajdova